vaultwarden/README.md

```diff
@@ -41,7 +45,7 @@ registration:
 
 ```
 # Allow public registrations, but don't forget to disable this again later.
-make enable-registration
+gmake enable-registration
```

Comment:
Is this supposed to be `gmake` or `make`?

Reply:
make :+1: